### PR TITLE
Update battleship_sem.py with bogus code

### PR DESCRIPTION
**Pull request by travissarles**
I wrote a random line of code which should be deleted.

### CODE REVIEW

File: battleship_sem.py [hunk 1]
```diff
@@ -12,6 +12,8 @@ def select_random_up_down():
     return random.choice(up_down)
 def select_random_left_right():
     return random.choice(left_right)
+def select_random_left_deleteme():
```
Comment (s-mehnert) [inline battleship_sem.py]: This code is terrible. please rewrite.

File: battleship_sem.py [hunk 2]
```diff
@@ -451,4 +453,4 @@ def battleship_game(play_count=0, won_count=0, lost_count=0):
     else:
         print(f"It was nice playing with you, {player1}! CU next time.")  
 
-battleship_game()
```
Comment (s-mehnert) [inline battleship_sem.py:454]: you broke the formatting, please check your formatter

**Review by s-mehnert — CHANGES_REQUESTED** [review]
we need to talk about this.

File: battleship_sem.py (hunk 2 above)
Comment (travissarles) [reply battleship_sem.py:454]: I'm so sorry. I'll fix it right away, boss!

**Review by s-mehnert — APPROVED** [review]
lgtm

**Pull request closed without merging**